tests/hmm/hmumu_utils.py

```diff
@@ -1667,6 +1681,84 @@ def compute_eff_product(offsets, jets_mask_passes_id, jets_eff, use_cuda):
         compute_eff_product_cpu(offsets, jets_mask_passes_id, jets_eff, p_puid)
     return p_puid
 
+
+@numba.njit(parallel=True)
+def compute_event_btag_weight_shape(offsets, eta_mask, jets_sf, out_weight):
```

Comment:
you can also optionally replace this function with a call to `ha.prod_in_offsets`, see [here](https://github.com/hepaccelerate/hepaccelerate/blob/master/hepaccelerate/backend_cpu.py#L248).

Reply:
So right now, the function is updated such that it doesn't even require the mask. The mask is applied before calling this function. Do you still think we should change it?